simln-lib/src/cln.rs

```diff
@@ -87,7 +87,8 @@ impl ClnNode {
             .map_err(|err| LightningError::GetInfoError(err.to_string()))?;
         connection.id.validate(&pubkey, &mut alias)?;
 
-        let features = if let Some(features) = our_features {
+        let features = if let Some(mut features) = our_features {
+            features.node.reverse();
             NodeFeatures::from_le_bytes(features.node)
```

Comment:
You should be able to use `NodeFeatures::from_be_bytes` if I'm not mistaken. I PRd a change to make that public on LDK a while ago but it looks like we forgot to apply the changes downstream

Reply:
Ah even better. I will switch both calls to use `from_be_bytes`. Thanks!